x/staking/module.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	consensusVersion uint64 = 4
+	consensusVersion uint64 = 3
```

Comment:
Explanation: we were at 2 in v0.45

https://github.com/cosmos/cosmos-sdk/blob/8bbeba5b46ff97c0f2d32226b56415b51ea9812e/x/staking/module.go#L164

No need to bump 2 versions at once